state-chain/pallets/cf-swapping/src/lib.rs

```diff
@@ -456,7 +456,8 @@ pub mod pallet {
 			if channel_metadata.is_some() {
 				// Currently only Ethereum supports CCM.
 				ensure!(
-					ForeignChain::Ethereum == destination_asset.into(),
+					ForeignChain::Ethereum == destination_asset.into() ||
+						ForeignChain::Arbitrum == destination_asset.into(),
```

Comment:
This should probably be refactored into a function/type that contains chains that support CCM, since in the future there will be more chains with CCM (e.g. Solana).